api/transactions/index.js

```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+const { Transactions } = require("../../db/Models");
+const createTransaction = require("../../helpers/Transactions/createTransaction");
+const loginMiddleware = require("../../middleware/checkLoggedIn");
+
+router.post("/", loginMiddleware, async(req, res) =>{
+  const { id, payer, status, amount, datePaid, appointmentId } = req.body;
```

Comment:
`payer` is not needed here yet. It should be nullable. `status` will be unpaid by default. `datePaid` will be automatically generated. Clients should not have to submit a date they paid.